python/pyspark/sql/tests/test_python_streaming_datasource.py

```diff
@@ -136,6 +137,33 @@ def streamWriter(self, schema, overwrite):
 
         return TestDataSource
 
+    def _get_simple_data_source(self):
```

Comment:
Maybe just define this class and directly pass `SimpleDataSource` to `register`?

Reply:
Sounds good.